service/storage/prefixes.go

```diff
@@ -14,7 +14,12 @@
 
 package storage
 
+// TODO: Reorder the prefixes so first has 1 and last 2, before we tag the first
+// release candidate:
+// => https://github.com/optakt/flow-dps/issues/134
+
 const (
+	prefixFirstHeight = 6
```

Comment:
Why not do it in this PR directly, since for now there's no production DPS server running and no one would be impacted?

Reply:
Because I would have to re-index the entire `mainnet-8`,  which takes 10+ hours, and `mainnet-7`, which takes a similar amount of time.